python/hsfs/feature_group.py

```diff
@@ -218,6 +218,27 @@ def get_feature(self, name: str):
                 f"'FeatureGroup' object has no feature called '{name}'."
             )
 
+    def compute_statistics(self):
```

Comment:
Can you please not mix the methods? The rest of the statistics methods start at line 265.